release/artifacts.yaml

```
@@ -143,7 +143,10 @@
   mavenUpload: true
 - target: //ledger/ledger-api-auth-client:ledger-api-auth-client
   type: jar-lib
-  mavenUpload: true
+  mavenUpload: True
```

```suggestion
  mavenUpload: true
```
Probably meaningless but if there's an established standard I'd stick with that. :wink: